CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ be found.
 
 ## 1.13.0 (2016-12-08)
 
+**IMPORTANT**: In Docker 1.13 managed plugin api changed from the experimental version introduced in 
+Docker 1.12. Once upgrading to Docker 1.13 you might having issues at daemon start:
+> Error starting daemon: json: cannot unmarshal string into Go value of type types.PluginEnv
+
+To resolve this:
+
+- Remove plugins.json from: `/var/lib/docker/plugins/`
+- Restart Docker
+- Reinstall your plugins
```

Comment:
I think uninstall and reinstall is a better suggestion, also because simply removing `plugins.json` is not sufficient. The plugins need to be removed from disk as well. So lets go with `please uninstall your plugins before upgrade and reinstall them after upgrade`